roles/rsyslog/defaults/main.yaml

```diff
@@ -948,36 +961,51 @@ rsyslog__conf_viaq_elasticsearch:
               } else {
                   set $.viaq_index_name = exec_template("prefix_index_template");
               }
-          {% if openshift_logging_use_ops | default(False) %}
-              if $.viaq_index_prefix startswith "project." then {
+          {% for res in rsyslog_elasticsearch %}
+          {% if loop.length == 1 %}
+              set $.es_host="{{ res.server_host | default('logging-es') }}"
+              set $.es_port="{{ res.server_port | default(9200) }}"
+            {% if use_omelasticsearch_cert | default(False) %}
+              set $.es_ca_cert="{{ res.ca_cert }}"
+              set $.es_cert="{{ res.cert }}"
+              set $.es_key="{{ res.key }}"
+            {% endif %}
+          {% else %}
+          {% if loop.first %}
+              if $.viaq_index_prefix startswith "{{ res.index_prefix }}" then {
+                set $.es_host="{{ res.server_host | default('logging-es') }}"
+                set $.es_port="{{ res.server_port | default(9200) }}"
+          {% if use_omelasticsearch_cert | default(False) %}
+                set $.es_ca_cert="{{ res.ca_cert }}"
+                set $.es_cert="{{ res.cert }}"
+                set $.es_key="{{ res.key }}"
           {% endif %}
+          {% elif loop.last %}
```

Comment:
What will happen in case user did not specify an index_prefix and it is not the last elasticsearch?
Do we want to fail in this case?

@richm @nhosoi @pcahyna

Reply:
@sradco yes